sstables/sstables.cc

```diff
@@ -3357,7 +3357,7 @@ future<file> sstable::s3_storage::open_component(const sstable& sst, component_t
 future<data_sink> sstable::s3_storage::make_data_or_index_sink(sstable& sst, component_type type, io_priority_class pc) {
     assert(type == component_type::Data || type == component_type::Index);
     co_await ensure_remote_prefix(sst);
-    co_return _client->make_upload_sink(make_s3_object_name(sst, type));
+    co_return _client->make_upload_jumbo_sink(make_s3_object_name(sst, type));
```

Comment:
add a FIXME comment saying that if we have "size upper bound' upfront, we can use make_upload_sink() instead?